Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqTable" {
+
+    It "should list tables from a dataset object from pipeline."{
+        $tables = Get-GcbqDataset “pshell_testing” | Get-GcbqTable
+        $tables.Count | Should Not Be 0
+    }
+
+    It "should list tables from a dataset object via parameter."{
+        $table = Get-GcbqDataset “pshell_testing” 
+        $tables = Get-GcbqTable -InputObject $table
+        $tables.Count | Should Not Be 0
+    }
+
+    It "should list tables from a dataset ID by parameter."{
+        $tables = Get-GcbqTable -Dataset “pshell_testing”
+        $tables.Count | Should Not Be 0
+    }
+
+    It "should get a singular table with a dataset object from pipeline."{
+        $table = Get-GcbqDataset “pshell_testing” | Get-GcbqTable "my_table"
+        $table.TableReference.TableId | Should be "my_table"
+        $table.Description | Should Be "TEST"
```

Comment:
Looking at your sample data, both tables have the same description. You should give them different descriptions.